### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by shwetabhandare**

### CODE REVIEW

File: src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java [hunk 1]
```diff
@@ -81,7 +80,7 @@ private ZMQRestResponse convert(RestResponse response) throws IOException {
 			zmqResponse.setContentType(response.contentType());
 		}
         if (response.content().length() > 0) {
-            if (response.contentThreadSafe()) {
+            if (zmqResponse.contentThreadSafe()) {
```
Comment (shwetabhandare) [inline src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java]: response.contentThreadSafe() function does not exist. So I had to use zmqRespose and that breaks the unit tests.

**Comment by KjellKod** [comment]
Can you add a license (ask Anton) so we can fork this to GitHub.com/LogRhythm?
@vrecan  is that OK Ben?

File: src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java (hunk 1 above)
Comment (KjellKod) [inline src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java]: Has it ever existed?

Comment (vrecan) [inline src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java]: yes or it would have never compiled

**Comment by vrecan** [comment]
:+1:

**Pull request merged**